UploadDaemon/SymbolAnalysis/SymbolCollection.cs

```diff
@@ -70,6 +72,11 @@ public SymbolCollection(List<AssemblyMethodMappings> mappings)
         /// </summary>
         public bool IsEmpty => mappings.Count() == 0 || mappings.Values.All(mapping => mapping.Count() == 0);
 
+        /// <summary>
+        /// The paths of the PDB symbol files considered in this collection.
+        /// </summary>
+        public ICollection<string> SymbolFileNames { get; private set; }
```

Comment:
Rename to SymbolFilePaths? Name is confusing for me as I expect this to be just the file name